tests/rptest/tests/partition_balancer_test.py

```diff
@@ -75,6 +75,65 @@ def node2partition_count(self):
 
         return ret
 
+    def get_partition_amount_on_node(self, node):
+        admin = Admin(self.redpanda)
+        return len(
+            list(
+                filter(lambda x: x["ns"] == "kafka",
+                       admin.get_partitions(node=node))))
+
+    def validate_metrics_node_down(self, disabled_node,
+                                   expected_moving_partitions_amount):
+        self.logger.debug(
+            f"Checking metrics for movement with disabled node: {self.redpanda.idx(disabled_node)}"
+        )
+        availabale_nodes = list(
+            filter(lambda x: x != disabled_node, self.redpanda.nodes))
+        metrics = self.redpanda.metrics_sample(
+            "moving_to_node",
+            availabale_nodes,
+            metrics_endpoint=MetricsEndpoint.PUBLIC_METRICS)
+        moving_to_node_partitions_amount = int(
+            sum([metric.value for metric in metrics.samples]))
+
+        self.logger.debug(f"\
+            moving_to_node_partitions_amount: {moving_to_node_partitions_amount}, \
+            expected_moving_partitions_amount: {expected_moving_partitions_amount}"
```

Comment:
would benefit from node id in the message

Reply:
Here we have the sum of values from all nodes.
Changed names